pkg/cmd/types.go

```diff
@@ -9,6 +9,19 @@ var (
 	ScanOptions = []string{"openscap"}
 )
 
+type MultiStringVar struct {
+	Values []string
+}
+
+func (sv *MultiStringVar) Set(s string) error {
```

Comment:
nit: This is more of an `add` than a `set`.  Have calls most of the morning but I'll take another look this afternoon.  If it's working and there are no other comments then don't wait on me. :+1: